README.md

```diff
@@ -145,13 +145,22 @@ module.exports = {
 }
 ```
 
-**Form submission triggers a `'submission'` [server-side event](https://v3.docs.apostrophecms.org/guide/server-events.html)** that you can listen for and handle in the `@apostrophecms/form` module or another module. Event handler functions are passed the following arguments:
+**First, form submission triggers a `'beforeSubmission'` [server-side event](https://v3.docs.apostrophecms.org/guide/server-events.html)** that you can listen for and handle in the `@apostrophecms/form` module or another module. Event handler functions are passed the following arguments:
+
+| Argument | Description |
+| ------- | ------- |
+| `req` | The request object from the submission |
+| `info` | An object with `form`, `data`, and `submission` properties |
```

Comment:
The arguments are:
* `submission`: `req`, `form`, `output`
* `beforeSaveSubmission`: `req`, `{ form, data, submission }`